tutor/src/flux/current-user.coffee

```diff
@@ -97,15 +105,23 @@ CurrentUserStore = flux.createStore
   _getParamsForRoute: (courseId, routeType, menuRole) ->
     if _.isFunction(ROUTES[routeType].params)
       ROUTES[routeType].params(courseId, menuRole)
+    else unless _.isUndefined(ROUTES[routeType].params)
+      ROUTES[routeType].params
     else
```

Comment:
would it be clearer to say `else if ROUTES[routeType].params?` vs `_.isUndefined` ?  I just mildly dislike `else unless` since I have to flip the logic around in my brain.